distribution/changelog.txt

```diff
@@ -78,6 +78,7 @@
 - Fix: [#16779] Fix case where title music doesn't unmute properly.
 - Fix: [#16808] Incorrect track design serialisation causing vehicle object replacement.
 - Fix: [objects#165] Glitch when Bengal Tiger Cars go through a corner.
+- Fix: [#16934] Park size displayed incorrectly in Park window.
```

Comment:
This should be moved up one line.

Reply:
Or it should be removed altogether, we're still waiting for Gymnasiast's reply

Reply:
Remove it. This bug isn't present in the last release so it doesn't need one.

Reply:
> Remove it. This bug isn't present in the last release so it doesn't need one.

Removed